### DISCUSSION TITLE
Plugin thumbnail images

### DISCUSSION THREAD

**Discussion by jeetee** [musescore/MuseScore#16077] [Ideas]
This question comes from looking at https://github.com/musescore/MuseScore/issues/16023 and having recently ported some of my own plugins to MS4 ([ABC import](https://musescore.org/project/abc-import), [NAF Fingering](https://musescore.org/en/project/native-american-flute-fingering)).

Could we have a series of consistently styled clean backgrounds for the different plugin categories? I tried to make mine somewhat follow the styling of the existing MuseScore provided ones, but I'm no graphic designer and some form of consistency would be nice.
Would this something you could provide @jessjwilliamson ?

Here's how my ports are currently looking:
![image](https://user-images.githubusercontent.com/895153/214806306-b6ddb465-b1d6-4289-bc76-aa294ffddf6e.png)

**Comment by Jojo-Schmitz** [comment]
Here's the one for Batch Convert (which doesn't really work yet in Mu4)
![batch_convert_thumbnail](https://user-images.githubusercontent.com/1786669/214807057-d887db9f-b099-4350-b246-8251e902bf1d.png)

**Comment by XiaoMigros** [comment]
Any update on this? It would be extremely helpful to have some degree of visual consistency for these thumbnails.

**Comment by MarcSabatella** [comment]
We might also consider some new categories.  "Composing/arranging tools" is big and generic enough that virtually everything could conceivably fit in it.  "Color notes" is so specific that there would not likely ever be more than a couple of plugins in that category.

**Reply by XiaoMigros** [reply]
I think adopting categories similar to the ones on the https://musescore.org/plugins page would make sense
Alternatively it might make sense to remodel the website's plugins page after what the program ends up going with
(Also it would be great if users could at least make their own subcategories, it's quite handy for groups of very similar plugins)

**Comment by XiaoMigros** [comment]
@jessjwilliamson how's this looking?

**Comment by jeetee** [comment]
For those with GIMP, I've got https://github.com/jeetee/MuseScore_TempoStretch/blob/master/asset/TempoStretch.xcf as my most recent iteration, having three category background gradients, a generic "file" icon template and a default quarter note copied from one of the built-in icons (rasterized, but not too bad looking at the smaller view size of the MuseScore plugin tab).
